### PR TITLE
WEB-1814 support course codes

### PR DESCRIPTION
**Pull request by TamoshaytisV**
https://liv-it.atlassian.net/browse/WEB-1814
Related API PR:
Students should be able to enroll by course codes the same way they do with vouchers.
For this reason, we use "Enter Voucher" page. "Voucher" is renamed to "Access code" everywhere to reflect the ability to accept several types of codes.
Could be merged without API part

### CODE REVIEW

File: lms/templates/labster/enter_voucher.html [hunk 1]
```diff
@@ -54,10 +54,10 @@ <h3 class="message-title">${_("The following errors occurred while processing yo
             <li class="field required text" id="field-name">
               <label for="code">${_('Code')}</label>
               <input id="code" type="text" name="code"
-                placeholder="${_('Please enter purchased voucher code')}"
+                placeholder="${_('Please enter your Voucher Code or Course Code')}"
                 required
                 aria-required="true" aria-describedby="code-tip"/>
-              <span class="tip tip-input" id="code-tip">${_("Needed for access to Labster courses")}</span>
+              <span class="tip tip-input" id="code-tip">${_("In case you have any questions please write us at support@labster.com")}</span>
```
Comment (TamoshaytisV) [inline lms/templates/labster/enter_voucher.html]: Text was provided by Emma

File: lms/templates/labster/enter_voucher.html [hunk 2]
```diff
@@ -54,10 +54,10 @@ <h3 class="message-title">${_("The following errors occurred while processing yo
             <li class="field required text" id="field-name">
               <label for="code">${_('Code')}</label>
               <input id="code" type="text" name="code"
-                placeholder="${_('Please enter purchased voucher code')}"
+                placeholder="${_('Please enter your Voucher Code or Course Code')}"
```
Comment (TamoshaytisV) [inline lms/templates/labster/enter_voucher.html:58]: Text was provided by Emma

File: lms/djangoapps/labster_vouchers/views.py [hunk 3]
```diff
@@ -99,12 +99,12 @@ def activate_voucher(request):
...
     except ItemNotFoundError:
         messages.error(request, _(
-            "Cannot find a voucher '{}'. Please contact Labster support team."
+            "Cannot find an access code '{}'. Please contact Labster support team."
         ).format(code))
         return redirect(enter_voucher_url)
     except LabsterApiError:
         messages.error(request, _(
-            "There are some issues with applying your voucher. Please try again in a few minutes."
+            "There are some issues with applying your acess code. Please try again in a few minutes."
```
Comment (polesye) [inline lms/djangoapps/labster_vouchers/views.py]: typo `acess` -> `access`

File: lms/templates/labster/enter_voucher.html (hunk 1 above)
Comment (polesye) [reply lms/templates/labster/enter_voucher.html]: use settings.SUPPORT_EMAIL if it exists

**Review by polesye — APPROVED** [review]

**Review by lukyfebrianto — APPROVED** [review]

**Pull request merged**